### PR TITLE
fix(thumbnails): missing field, logging and new config var

### PR DESCRIPTION
**Pull request by dpgaspar**
### SUMMARY
PR with a couple of minor fixes:

- Add missing `thumbnail_url` to the list chart API
- Fixes chart thumbnail logging
- Introduces new CONFIG key, distinct from the email report key

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [X] Removes existing feature or API: Removes EMAIL_REPORTS_WEBDRIVER config key

### CODE REVIEW

File: superset/charts/api.py [hunk 1]
```diff
@@ -86,57 +86,58 @@ class ChartRestApi(BaseSupersetModelRestApi):
...
         "datasource_name_text",
+        "datasource_type",
         "datasource_url",
+        "description",
+        "id",
+        "params",
+        "slice_name",
         "table.default_endpoint",
         "table.table_name",
+        "thumbnail_url",
```
Comment (dpgaspar) [inline superset/charts/api.py:120]: sorted and added a new field `thumbnail_url`

File: superset/config.py [hunk 2]
```diff
@@ -797,6 +797,8 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 
 # Any config options to be passed as-is to the webdriver
 WEBDRIVER_CONFIGURATION: Dict[Any, Any] = {}
+# The webdriver to use supports "chrome" and "firefox"
+WEBDRIVER_TYPE = "firefox"
```
Comment (bkyryliuk) [inline superset/config.py:801]: let's use EMAIL_REPORTS_WEBDRIVER or just converge those 2
easy solution would be:
```
EMAIL_REPORTS_WEBDRIVER = EMAIL_REPORTS_WEBDRIVER
```

**Review by bkyryliuk — APPROVED** [review]

File: superset/config.py (hunk 2 above)
Comment (dpgaspar) [reply superset/config.py]: Maybe took it too far, but I actually removed it now. Tell me if it's ok by you

**Review by bkyryliuk — APPROVED** [review]

**Review by mistercrunch — APPROVED** [review]

File: superset/config.py [hunk 3]
```diff
@@ -790,7 +790,7 @@ class CeleryConfig:  # pylint: disable=too-few-public-methods
 # chrome:
 #   Requires: headless chrome
 #   Limitations: unable to generate screenshots of elements
-EMAIL_REPORTS_WEBDRIVER = "firefox"
+WEBDRIVER_TYPE = "firefox"
```
Comment (mistercrunch) [inline superset/config.py:793]: optional: change default to `"chrome"` ?

Comment (dpgaspar) [reply superset/config.py:793]: I would, but: https://github.com/apache/incubator-superset/blob/master/docs/installation.rst#email-reports

States that `firefox` is preferred, is it reasonable to change the default?

Comment (villebro) [reply superset/config.py:793]: I believe there are unresolved rendering errors on Firefox. Might not be applicable for thumbs, but worth mentioning.

**Review by villebro — APPROVED** [review]
LGTM

**Pull request merged**